appGeo/src/app/player/player.component.ts

```diff
@@ -28,7 +28,7 @@ export class Player implements OnInit{
     votes: number // An int
     chat_lists // List of Chat Objects that Player is a part of
     isAdmin : boolean // boolean whether this player is an admin (as in responsible for running the game start function)
-    
+    isKiller: boolean // boolean whether this player is a killer
```

Comment:
What's the point in this as opposed to using instanceOf?

Reply:
> What's the point in this as opposed to using instanceOf?

Jose and I talked about how it would be more convenient to just have a boolean field whether a player is a killer or not over making another subclass of a killer.
Basically the killing function will execute when the kill button is pressed (or from any other input from the front-end element), and at the button click we can just check whether the player has the right to kill or not. @jasulcaf Feel free to add any of your comment.

Reply:
Due to using Firebase the type can be destroyed and then instanceOf wouldn't work. To ensure, I'll just do both as a precaution